tests/test_grounding.py

```diff
@@ -51,7 +56,12 @@ def test_terminal_grounder(self):
         def mock_input(prompt):
             return next(mock_input_gen)
 
-        with mock.patch('__builtin__.raw_input', mock_input):
```

Comment:
Sorry, my bad, it's `six.moves.input`
https://pythonhosted.org/six/#module-six.moves
It is understandable as is, but if we can avoid having to support 2/3 compat ourselves, so much the better, it'll be less error prone